docs/VARIABLES.md

```diff
@@ -332,11 +332,19 @@ Default:  "{{ archive_destination_path }}"
 
 ***
 
-### config_prefix
+### ### Note - Only valid to customize when installation_method
```

Comment:
Not sure I understand this statement, what should installation_method be?

Reply:
The mail.yaml file has two lines of ### markers, causing this kind of doc generation. I have merged them in one line. however, I feel we need to enhance the script to accept multiline comments as well.

Reply:
Hey, I have a similar doubt. What does this statement mean? - `Only valid to customize when installation_method`
Sounds incomplete

Reply:
These were because description(###) was in two subsequent lines, which is not handled by our script currently. I have modified the comments and should be good in latest changes.